packages/locale/res/translations/tw.json

```diff
@@ -2155,4 +2155,4 @@
     "Custom",
     "自訂"
   ]
-]
\ No newline at end of file
+]
```

Comment:
@rhuba8324 can you try not adding this EOL, or running `npm run locale:extract` from the root of the repo to automatically fix it? It's fine if you can't, I'll just have to fix it after merging.

Reply:
It's now fixed, at least I think so.
Sorry for my mistake.

Reply:
No worries, not something you should know _a priori_!